docs/site/package.json

```diff
@@ -13,21 +13,21 @@
     "clean": "rimraf build"
   },
   "devDependencies": {
-    "babel-cli": "^6.18.0",
+    "babel-cli": "^6.22.2",
```

Comment:
What are your thoughts on only declaring a specific minimum minor and patch where there is an explicit dependency on them? `^6.0.0`, `^6.18.0` & `^6.22.2` are inherently the same as I understand it - all will install the latest 6.x.x release.

However, restricting it to the very latest versions (`^x.x.x`) risks a version conflict for some codependency that for whatever reason has pinned its dependancy below the latest.

Leaving all but those we need to push to a higher minor / patch at `^x.0.0` makes for a cleaner `package.json`, and makes it clear which ones have a specific minimum.